INSTALL.md

```diff
@@ -129,6 +129,25 @@ If you keep your site on another branch, open `.github/workflows/deploy.yml` **o
 
 If you need to manually re-deploy your website to GitHub pages, go to Actions, click "Deploy" in the left sidebar, then "Run workflow."
 
+### Deploy on [Netlify](https://www.netlify.com/)
+
+1. [Use this template -> Create a new repository](https://github.com/new?template_name=al-folio&template_owner=alshedivat).
+2. Netlify: **Add new site** -> **Import an existing project** -> **GitHub** and give Netlify access to the repository you just created.
+3. Netlify: In the deploy settings
+    - Set **Branch to deploy** to `master`
```

Comment:
The branch here should be `main`.

Reply:
> The branch here should be `main`.

I see. Thank you for pointing that out. It's now fixed.